Makefile

```
@@ -8,38 +8,16 @@ INITRAMFS_CPIO = $(OUTPUT_DIR)/initramfs.cpio
-	@echo "$(C_DIM)│$(C_RESET)  $(C_CYAN)clean$(C_RESET)          $(C_DIM)│$(C_RESET) Clean build artifacts                              $(C_DIM)│$(C_RESET)"
-	@echo "$(C_DIM)│$(C_RESET)  $(C_CYAN)clean-downloads$(C_RESET)$(C_DIM)│$(C_RESET) Remove downloaded binaries (sgdisk, dd)            $(C_DIM)│$(C_RESET)"
-	@echo "$(C_DIM)│$(C_RESET)  $(C_CYAN)clean-all$(C_RESET)      $(C_DIM)│$(C_RESET) Run both clean and clean-downloads                 $(C_DIM)│$(C_RESET)"
-	@echo "$(C_DIM)│$(C_RESET)  $(C_CYAN)vm$(C_RESET)             $(C_DIM)│$(C_RESET) Run the built ISO in a QEMU VM                     $(C_DIM)│$(C_RESET)"
-	@echo "$(C_DIM)│$(C_RESET)  $(C_CYAN)dep$(C_RESET)            $(C_DIM)│$(C_RESET) Check for dependencies                             $(C_DIM)│$(C_RESET)"
-	@echo "$(C_DIM)╰──────────────────────────────────────────────────────────────────────╯$(C_RESET)"
-	@echo ""
+	@echo "run 'make' to compile"
+	@echo "does not support -j$(nproc)"
+	@echo "on fedora, run 'make -f Makefile-fedora'"
 
 dep:
```

checkmake (0.2.2)

[warning] Target body for "dep" exceeds allowed length of 5 (10).

(maxbodylength)